### ISSUE TITLE
Insert time in headline now inserts time in body

### ISSUE THREAD

**Issue by SegundoBob** [leo-editor/leo-editor#131]
I first noticed this 2014-12-03 Wed in Revision ad0392f8b69824cf19441f9ff9636e654341d536.
It still happens in commit 2014-01-16 07:13 f6cdb2bccb1a7ca8812f49f0da951cec5617279f.

Demo:
```
Ctrl-I
Alt-E, h, t
```

Test system
Xubuntu32 12.04
```
Python 2.7.3, PyQt version 4.8.1
and
Python 3.2.3, PyQt version 4.8.1
```

**Comment by technatica** [comment]
This happens for me as well.

Leo 5.1-final, build 20150419062316, Sun Apr 19 06:23:16 CDT 2015
Git repo info: branch = master, commit = 4243ee509d73
Python 2.7.6, PyQt version 4.8.6
linux2

Chris

**Comment by edreamleo** [comment]
Fixed at rev c6aa962.  Please reopen if there are further problems.
